Formula/geocouch.rb

```diff
@@ -1,11 +1,10 @@
 class Geocouch < Formula
```

Comment:
I don't think so, at least not yet. For users who have `geocouch` installed already, we should give them a ready way to see info about this breaking change. Because they won't have `couchdb` installed `--with-geocouch`, the GeoCouch-related caveats will not be displayed for `couchdb`, and they may not even know to look there.

Also, I think if we just remove the formula, than kegs for existing installations of it will be left in place when a `brew update; brew upgrade` is done, right? We want the files installed by `geocouch` to actually be removed, so they don't conflict with `couchdb`. I don't think that will happen with the way the new `couchdb --with-geocouch` is written, but I'm not sure, especially if the user has set up `$ERL_LIBS` based on the earlier installation. Forcing the removal as part of `brew upgrade` seems less likely to leave a user with an unsupported in-between state in their brew prefix.

Once it's been long enough that all GeoCouch users have gone through a `brew update; brew upgrade` cycle and are now on `couchdb --with-geocouch`, then I think `geocouch` can be removed.